tests/test_authentication_jwt.py

```diff
@@ -68,9 +68,9 @@ async def test_valid_token(self, jwt_authentication, mock_user_db, token, user):
 async def test_get_login_response(jwt_authentication, user):
     login_response = await jwt_authentication.get_login_response(user, Response())
 
-    assert "token" in login_response
+    assert "access_token" in login_response
```

Comment:
We should add `assert login_response["token_type"] == "bearer"` here

Reply:
Sure, I'll do during the weekend